rakelib/kokoro/kokoro_builder.rb

```diff
@@ -45,6 +43,13 @@ def build_kokoro_configs
                         "./.kokoro/#{build_type}/#{os_version}/#{name}.cfg",
                         gem: gem
         end
+        next unless name =~ /-v\d/
+
+        [:samples_latest, :samples_master].each do |build_type|
+          from_template "./.kokoro/templates/#{os_version}.cfg.erb",
+                        "./.kokoro/#{build_type}/#{os_version}/#{name}.cfg",
+                        gem: gem
+        end
```

Comment:
Nit: instead of the repetition, I think it would be a bit more clear if, after determining the name, you say something like:
```ruby
build_types = [:continuous, :nightly]
build_types += [:samples_latest, :samples_master] if name =~ /-v\d\w+$/
# Now you just need one copy of the inner loop
```

(I also think we can tighten the regex, as illustrated above.)

Reply:
good call on the simplifying the loop. that regex doesn't work for "-v1". it's separated from "google-cloud-" so I think even just checking for a "-" would be fine 🤷‍♂

Reply:
oh right, should be `/-v\d\w*$/`

Checking for just `-` won't work because of `bigquery-data_transfer` and similar.